PointOfDispensingApp/src/components/PatientLookupPage.vue

```diff
@@ -120,14 +139,24 @@
 <script>
   export default {
     name: 'PatientLookupPage',
+    watch: {
+      date () {
+        this.dateFormatted = this.formatDate(this.date)
+      },
+    },
+    computed: {
+      computedDateFormatted () {
+        return this.formatDate(this.date)
+      },
+    },
     methods: 
     {
       rowClick: function (item, row) {      
         row.select(true);
         this.selectedId=item.id
         this.selectedFamilyName=item.familyName
         this.selectedGivenName=item.givenName
-        this.selectedDOB=item.DOB
+        this.selectedDOB=item.date
```

Comment:
```suggestion
        this.selectedDOB=item.DOB
```
item.date causes issues because the objects in patientLookupTable dont have a date field. These lines have been removed though recently (in Development).

Reply:
Change isnt necessary since the lines have been removed